### DISCUSSION TITLE
Homebrew with non sudo user on macOS

### DISCUSSION THREAD

**Discussion by mia0x1** [Homebrew/discussions#786] [Getting started] [answered]
hello,

I want to use Homebrew with a non sudo account on macOS. 
In order to run the install script I have to switch to my admin/sudo user. What do I have to do in order to run brew commands with another user which isn't the admin? Thx for your help :)

**Comment by SMillerDev** [comment, marked as answer]
You'll have to make sure your user is the owner of all files brew installed.

**Reply by mia0x1** [reply]
Yes, thank you.
chown -R user:group /opt/homebrew is the solution if this is the homebrew folder.
